in_command.go

```diff
@@ -66,17 +66,19 @@ func (c *InCommand) Run(destDir string, request InRequest) (InResponse, error) {
 			return InResponse{}, err
 		}
 
-		commitPath := filepath.Join(destDir, "commit_sha")
-		commitSHA, err = c.resolveTagToCommitSHA(*foundRelease.TagName)
-		if err != nil {
-			return InResponse{}, err
-		}
-
-		if commitSHA != "" {
-			err = ioutil.WriteFile(commitPath, []byte(commitSHA), 0644)
+		if !request.Source.Drafts {
```

Comment:
This feels somewhat unintuitive to me, as it checks a very high-level configuration to know how to deal with the release, rather than looking at the release itself. Is there a way to check directly whether the release is a draft, rather than checking if the resource is configured with `drafts: true`? That would feel a bit more direct.

Reply:
Ah, that's true. Didn't catch it